releasenotes/notes/asm-subprocess-executions-479ab8fb59d9f312.yaml

```diff
@@ -0,0 +1,12 @@
+---
+prelude: >
+  Application Security Management (ASM) has added support for tracing subprocess executions.
+
+features:
+  - |
+    ASM: add support for tracing subprocess executions (like `os.system`, `os.spawn`, `subprocess.Popen` and others) and adding
```

Comment:
We try to avoid using release notes as the primary source for documentation. Ideally a release note should just call out a change and link to a doc for more information. 

Since this is large high impact feature I cam okay with this descriptive release note. In a follow up PR can we start documenting more ASM features and configurations in our ddtrace library docs: 
Ex:
https://github.com/DataDog/dd-trace-py/blob/bb5dccc1405723d103a132475ac00ea693cdafcd/docs/api.rst#api
https://github.com/DataDog/dd-trace-py/blob/bb5dccc1405723d103a132475ac00ea693cdafcd/docs/index.rst
https://github.com/DataDog/dd-trace-py/blob/bb5dccc1405723d103a132475ac00ea693cdafcd/docs/configuration.rst

Reply:
Thanks. Yes, we usually defer to the global docs for ASM features which have Python-specific tabs but improving the dd-trace-py specific documentation would be great, I added a task to our Jira board so we can tackle it on some next sprint.